aidants_connect_web/urls.py

```diff
@@ -124,14 +124,19 @@
     ),
     path(
         "creation_mandat/visualisation/final/",
-        mandat.attestation_final,
+        mandat.AttestationTemplateView.as_view(),
         name="new_attestation_final",
```

Comment:
Ici un truc m'inquiète : en finissant la refacto vers CBV, je me suis finalement retrouvé avec 2 vues exactement identiques pour `new_attestation_final` et `new_attestation_projet`. J'ai l'impression d'avoir loupé un truc mais même en relisant bien la refacto, j'arrive pas à savoir où sont les différences…

Reply:
En y repensant, ça semble pas déconnant : la seule différence entre le projet et l'attestation finale, c'est le QR code ! Je crois qu'il y a même pas de message pour expliquer que l'un est juste un projet. En tout cas pas dans le template.

Reply:
Alors non c'est pas la même. 

Deux différence déjà : 
- dans le template attestation_projet il y a en dur : "Vous trouverez ci-dessous le projet de mandat que vous êtes en train de créer." or si pour new_attestation_projet c'est bon pour le mandat en lui meme du coup c'est plus un projet du tout
 - ensuite dans le template du mandat il y a un `{% if not final %}
      <div class="navbar__gouvfr float-right">
        <i>Projet de mandat</i>
      </div>
    {% else %}
      <div class="navbar__gouvfr float-right">
        <img alt="" class="navbar__gouvfr img-qrcode" src="{% url 'new_attestation_qrcode' %}"/>
      </div>
    {% endif %}`

final étant mis a True dans new_attestation_final et pas dans new_attestation_projet